v2_1/ws/rest/src/sdmx-rest.yaml

```diff
@@ -169,7 +187,9 @@ paths:
 
   /metadata/metadataset/{providerID}/{resourceID}/{version}:
```

Comment:
Let's see if we really later need this, because if metadata is considered being similar to data rather than structures, then metadataset identification should not be needed. Ref metadata can already be retrieved through the /metadata/metadataflow/ query.

Reply:
Same reply as for point 1 above :)